docs/docs/guides/deploy/linux.mdx

```diff
@@ -19,8 +19,14 @@ cockroach start-single-node --insecure --background --http-addr :9090 --listen-a
 
 ## Install ZITADEL
 
+Download the ZITADEL release according to your architecture from [Github](https://github.com/zitadel/zitadel/releases/latest) 
+
+## Unpack the archive
+move to your download location and unpack the archive
+
 ```bash
-curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0 | grep "browser_download_url.*zitadel_Linux_$(uname -m).tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+#unpack and set executable
+tar xfv zitadel_* && chmod +x zitadel
```

Comment:
Please create a command that downloads and creates a usable binary

Reply:
done